examples/window/low_power.rs

```diff
@@ -70,7 +70,7 @@ fn update_winit(
 
 /// Switch between update modes when the mouse is clicked.
 fn cycle_modes(mut mode: ResMut<ExampleMode>, mouse_button_input: Res<Input<MouseButton>>) {
-    if mouse_button_input.just_pressed(MouseButton::Left) {
+    if mouse_button_input.just_pressed(MouseButton::Right) {
```

Comment:
This makes it a little more difficult to use on laptops perhaps.

Reply:
Well, I am speculating people using laptops won't find it difficult because they are used to it, there is always either a physical rmb or a two-finger tap. Curious to know why change it to right mouse button at all.

Reply:
I changed it because I found when trying out the demonstrated features - namely while defocusing and refocusing the window - I would inadvertently switch modes as I clicked on the window to bring it into focus.

I could change to spacebar to make it easier to switch intentionally.

Reply:
Oh, it is well-justified then. 
Anything else than LMB is fine.